doc/architecture/decisions/0019-BLAST-database-files.md

```diff
@@ -0,0 +1,41 @@
+# 19. BLAST database files
+
+Date: 2022-09-11
+
+## Status
+
+Draft
+
+## Context
+
+This document summarises the decisions made when choosing which database options to support for the initial implementation of BLAST on the new Ensembl website.  
+
+## Decision
+
+### Database choices
+BLAST databases are generated from FASTA files copied from the Ensembl and ensemblgenomes FTP sites
+
+#### Genomic sequence
+Unmmasked and softmasked masked, excluding hardmasked. Softmasked sequence has all the sequence, and so the coords are consistent, it’s just that parts are lowercase and will be ignored by BLAST when it’s looking for regions to seed the alignments. An alignment seed can be found near a repeat and the alignment can then be extended into the repeat though. This is based on the idea that you’re probably not looking for things that are wholly contained in repeats. Plus it makes BLAST much faster as it looks at less of the genome.
```

Comment:
> Unmmasked and softmasked masked, excluding hardmasked.

Aren't there too many "masked" in "softmasked masked"?

Also, doesn't the next sentence want to start a new paragraph?